src/admin/utils/fieldTypes.jsx

```diff
@@ -64,6 +64,11 @@ export const common = {
             title: "Collapsible",
             type: "boolean",
           },
+          hidden: {
```

Comment:
We should pass uiOptions to set this to a switch for consistency with the rest of the settings and change title to simply "Hidden".